src/Components/App/App.js

```diff
@@ -28,9 +34,13 @@ class App extends Component {
   }
 
   render() {
```

Comment:
I think maybe having a loading component would help with the issues with compenentDidMount giving us our default state when our component has not finished mounting.